edx/analytics/tasks/reports/payment.py

```diff
@@ -14,14 +15,18 @@ class PaymentTask(luigi.WrapperTask):
     )
 
     def requires(self):
-        yield PaypalTransactionsIntervalTask(
+        # Temporarily disable existing tasks to be able to run PaymentTask
+        # yield PaypalTransactionsIntervalTask(
+        #     interval_end=self.import_date
+        # )
+        # for merchant_id in self.cybersource_merchant_ids:
+        #     yield IntervalPullFromCybersourceTask(
+        #         interval_end=self.import_date,
+        #         merchant_id=merchant_id
+        #     )
```

Comment:
As a reminder, this needs to be uncommented before merging.  (Probably better to keep this as an uncommited local change to avoid the risk of merging it like this.)